### PR TITLE
Fix #49: Update byte stream format registry and spec headers, sotd, bug links, etc.

### LINKED ISSUE #49: Byte Stream Format pages still have "file a bug" buttons that link to Bugzilla
These buttons need to be removed like they were for the main spec.

### PR DESCRIPTION
**Pull request by wolenetz**
Includes previously missing sotd and abstract sections where necessary,
and removes unnecessary empty "SourceBuffer" entries in the local
respec definitionMap to net 0 respec errors for the updated respec
files in this change.

### CODE REVIEW

**Comment by wolenetz** [comment]
@jdsmith3000 Please take a look. I highly recommend we first get https://github.com/w3c/media-source/pull/117 reviewed and landed, then this (since #117 contains editor changes and overlapping respec regeneration output changes that are duplicated here).

**Comment by jdsmith3000** [comment]
Looks great.  Okay to merge.

I thought a bit about populating the Implementation header fields with generic MSE links, but think that adds little value for these registry specs.

**Comment by wolenetz** [comment]
Thanks for the review. I'll merge this manually shortly.

File: isobmff-byte-stream-format-respec.html [hunk 1]
```diff
@@ -114,6 +136,11 @@
...
     </section>
 
+    <section id="sotd">
+      <p>The working group maintains <a href="https://github.com/w3c/media-source/issues">a list of all bug reports that the editors have not yet tried to address</a>; there may also be open bugs in the <a href="http://w3.org/brief/Mjcw">previous bug tracker</a>.</p>
+      <p>Implementors should be aware that this specification is not stable. <strong>Implementors who are not taking part in the discussions are likely to find the specification changing out from under them in incompatible ways.</strong> Vendors interested in implementing this specification before it eventually reaches the Candidate Recommendation stage should join the mailing list mentioned below and take part in the discussions.</p>
+    </section>
+
     <section id="introduction">
       <h2>Introduction</h2>
       <p>This specification defines segment formats for implementations that choose to support the ISO Base Media File Format
```
Comment (wolenetz) [inline isobmff-byte-stream-format-respec.html]: Hmm. This file includes a revision history, so I'll update that as part of merging this PR.

**Pull request merged**